node/pkg/processor/message.go

```diff
@@ -120,3 +123,16 @@ func (p *Processor) handleMessage(k *common.MessagePublication) {
 		timeToHandleObservation.Observe(float64(time.Since(start).Microseconds()))
 	}
 }
+
+func (p *Processor) tssSetup(v *VAA) {
+	v.Version = vaa.TSSVaaVersion
+	digest := v.SigningDigest()
+	hash := hex.EncodeToString(digest.Bytes())
+
+	p.tssWaiters[hash] = timedThresholdSignatureWaiter{
```

Comment:
consider making tssWaiters a map of []byte -> timedThresholdSignatureWaiter, this way you wouldn't need to convert a digest to hex string and work with two representations.

Reply:
Since Go doesn't support `==` and `!=` operations between slices, it's impossible to use them as keys in maps. 
I chose to follow other examples from their code to create the keys of the `tssWaiters` map. (see processor/message.go lines 73, 99).